modules/exploits/windows/misc/regsvr32_applocker_bypass_server.rb

```diff
@@ -0,0 +1,101 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'msf/core/exploit/powershell'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Exploit::Powershell
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'         => 'Regsvr32.exe (.sct) Application Whitelisting Bypass Server',
+      'Description'  => %q(
+        This module simplifies the Regsvr32.exe Application Whitelisting Bypass technique.
+        The module creates a web server that hosts an .sct file. When the user types the provided regsvr32
+        command on a system, regsvr32 will request the .sct file and then execute the included PowerShell command.
+        This command then downloads and executes the specified payload (similar to the web_delivery module with PSH).
+        Both web requests (i.e., the .sct file and PowerShell download and execute) can occur on the same port.
+      ),
+      'License'      => MSF_LICENSE,
+      'Author'       =>
+        [
+          'Casey Smith @subTee', # AppLocker bypass research and vulnerability discovery
```

Comment:
I'd move handles into the comment. ```msftidy``` actually checks for this, but it's apparently (still?) broken atm.